src/main/java/cz/muni/fi/pv286/mmt/model/Options.java

```diff
@@ -0,0 +1,163 @@
+package cz.muni.fi.pv286.mmt.model;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Optional;
+
+/**
+ * This class encapsulates all possible options.
+ */
+public class Options {
+    private IoFormat inputFormat;
+    private Optional<IoOption> inputFromToOption = Optional.empty();
+    private IoFormat outputFormat;
+    private Optional<IoOption> outputFromToOption = Optional.empty();
+    private InputStream inputFile = System.in;
+    private OutputStream outputFile = System.out;
+    private String delimiter = "\n";
```

Comment:
**issue:** This is not a cross-platform delimiter.

This will not work on all platforms.